### DISCUSSION TITLE
easy swap node option

### DISCUSSION THREAD

**Discussion by tampadesignr** [comfyanonymous/ComfyUI#5887] [Ideas]
easy swap node option

example

comfyui updates and breaks something
you find the same node that is now red on workflow in nodes panel
you drop the good node next to the red node
click on red node with an option to swap with another node on stage
choose the one you dropped next to the red
and select it
quick on to the next thing

**Comment by ltdrdata** [comment]
Pleaes move this request to front repo.
https://github.com/Comfy-Org/ComfyUI_frontend

**Reply by tampadesignr** [reply]
done
